lib/cext/ruby.h

```diff
@@ -78,7 +78,7 @@ extern "C" {
 
 // Basic types
 
-typedef void *VALUE;
+typedef void* VALUE;
 typedef long SIGNED_VALUE;
```

Comment:
That's the way I've always seen it written. Compare `typedef void*` and `typedef void *` in MRI - the second one is the way they do it most of the time.